### ISSUE TITLE
`excessLoss` returns non-zero value when there is no loss

### ISSUE THREAD

**Issue by sjuanati** [groLabs/GSquared#80]
Field `excessLoss` in event `LogStrategyHarvestReport` from GVault is expected to return the difference between the previous lockedProfit and the loss if such a loss is greater than the previous lockedProfit:

```excessLoss = loss - lockedProfitBeforeLoss if loss > lockedProfitBeforeLoss```

For instance, if we had lockedProfitBeforeLoss= 50 and loss = 70 from the latest harvest event, we would get:
lockedProfit = 0
excessLoss = 20

However, in the harvest event at tx `0x49f54b33bbe0e88a0fbc5866185071134dc8ab80d706d9dd5c3ea24dfb12b132`, excessLoss has the same value as lockedProfit when there was no loss:

lockedProfit : 10946105586785600163
excessLoss : 10946105586785600163
